consts/estimate.ts

```diff
@@ -0,0 +1,4 @@
+// Account for gas estimation
+// will always have >=0.001 ether, >=0.001 stETH, >=0.001 wstETH
+// on Mainnet, Rinkeby, Goerli
+export const ESTIMATE_ACCOUNT = '0x87c0e047F4e4D3e289A56a36570D4CB957A37Ef1';
```

Comment:
I think you shouldn't use such things in the template

Reply:
think